tests/arrow_tests.py

```diff
@@ -1221,6 +1221,129 @@ def test_years(self):
+        assertEqual(result, arrow.Arrow(2013,1,1,0,0,1))
+
+        result = arw.dehumanize('1 second ago')
+        assertEqual(result, arrow.Arrow(2012,12,31,23,59,59))
+
+        result = arw.dehumanize('2 seconds later')
+        assertEqual(result, arrow.Arrow(2013,1,1,0,0,2))
+
+        result = arw.dehumanize('2 seconds ago')
+        assertEqual(result, arrow.Arrow(2012,12,31,23,59,58))
+
+    def test_minutes(self):
+        arw = arrow.Arrow.fromdatetime(self.datetime)
+
+        result = arw.dehumanize('1 minute later')
+        assertEqual(result, arrow.Arrow(2013,1,1,0,1,0))
+
+        result = arw.dehumanize('1 minute ago')
+        assertEqual(result, arrow.Arrow(2012,12,31,23,59,0))
+
+    def test_hours(self):
+        arw = arrow.Arrow.fromdatetime(self.datetime)
+
+        result = arw.dehumanize('1 hour later')
+        assertEqual(result, arrow.Arrow(2013,1,1,1,0,0))
+
+        result = arw.dehumanize('1 hour ago')
+        assertEqual(result, arrow.Arrow(2012,12,31,23,0,0))
+
+    def test_days(self):
+        arw = arrow.Arrow.fromdatetime(self.datetime)
+
+        result = arw.dehumanize('1 day later')
+        assertEqual(result, arrow.Arrow(2013,1,2,0,0,0))
+
+        result = arw.dehumanize('1 day ago')
+        assertEqual(result, arrow.Arrow(2012,12,31,0,0,0))
+
+    def test_weeks(self):
+        arw = arrow.Arrow.fromdatetime(self.datetime)
+
+        result = arw.dehumanize('1 week later')
+        assertEqual(result, arrow.Arrow(2013,1,8,0,0,0))
+
+        result = arw.dehumanize('1 week ago')
+        assertEqual(result, arrow.Arrow(2012,12,25,0,0,0))
+
+    def test_months(self):
+        arw = arrow.Arrow.fromdatetime(self.datetime)
+
+        result = arw.dehumanize('1 month later')
+        assertEqual(result, arrow.Arrow(2013,2,1,0,0,0))
+
+        result = arw.dehumanize('1 month ago')
+        assertEqual(result, arrow.Arrow(2012,12,1,0,0,0))
+
+    def test_years(self):
+        arw = arrow.Arrow.fromdatetime(self.datetime)
+
+        result = arw.dehumanize('1 year later')
+        assertEqual(result, arrow.Arrow(2014,1,1,0,0,0))
+
+        result = arw.dehumanize('1 year ago')
+        assertEqual(result, arrow.Arrow(2012,1,1,0,0,0))
+
+    def test_additional(self):
+        arw = arrow.Arrow.fromdatetime(self.datetime)
+        arw2012 = arrow.Arrow.fromdatetime(self.datetime2012)
+
+        result = arw.dehumanize('3 year 12 months 365 days later')
+        assertEqual(result, arrow.Arrow(2018,1,1,0,0,0))
+
+        result = arw2012.dehumanize('3 year 12 months 365 days later')
+        assertEqual(result, arrow.Arrow(2016,12,31,0,0,0))
+
+        result = arw.dehumanize('3 year 12 months 365 days ago')
+        assertEqual(result, arrow.Arrow(2008,1,2,0,0,0))
+
+        result = arw2012.dehumanize('3 year 12 months 365 days ago')
+        assertEqual(result, arrow.Arrow(2007,1,1,0,0,0))
+
+    def test_errors(self):
+        arw = arrow.Arrow.fromdatetime(self.datetime)
+
+        test = True
```

Comment:
You can't test with try-catch, since you don't know which Exception will be raisen.

use assertRaise instead.

```
class MyTestCase(unittest.TestCase):
    def test1(self):
        self.assertRaises(SomeCoolException, mymod.myfunc)
```

see also https://docs.python.org/dev/library/unittest.html#unittest.TestCase.assertRaises